sway-core/src/asm_generation/fuel/programs/abstract.rs

```diff
@@ -164,13 +164,13 @@ impl AbstractProgram {
     /// Right now, it looks like this:
     ///
     /// WORD OP
-    /// 1    MOV $scratch $pc
-    /// -    JMPF $zero i2
-    /// 2    DATA_START (0-32) (in bytes, offset from $is)
-    /// -    DATA_START (32-64)
-    /// 3    LW $ds $scratch 1
-    /// -    ADD $ds $ds $scratch
-    /// 4    .program_start:
+    /// [1]    MOV $scratch $pc
+    /// [-]    JMPF $zero i2
+    /// [2]    DATA_START (0-32) (in bytes, offset from $is)
+    /// [-]    DATA_START (32-64)
+    /// [3]    LW $ds $scratch 1
+    /// [-]    ADD $ds $ds $scratch
+    /// [4]    .program_start:
```

Comment:
I couldn't get this to render, so I tried the following which seemed to workm quite nicely:

```
    /// |OFFSET|WORD OP                                      |
    /// |------|---------------------------------------------|
    /// |1     |MOV $scratch $pc                             |
    /// |1.5   |JMPF $zero i2                                |
    /// |2     |DATA_START (0-32) (in bytes, offset from $is)|
    /// |      |DATA_START (32-64)                           |
    /// |3     |LW $ds $scratch 1                            |
    /// |3.5   |ADD $ds $ds $scratch                         |
    /// |4     |.program_start:                              |
```